### DISCUSSION TITLE
Information about pre-existing condition of patients?

### DISCUSSION THREAD

**Discussion by latlatina** [MIT-LCP/mimic-code#1617] [MIMIC-III]
Hi,
Is there any information regarding pre-existing diagnoses of the patients (such as cancer, CF, etc...).
If so, on which table and field can I find it?
Thanks.

**Comment by alistairewj** [comment]
There are a few PMH items in chart events, particularly for cardiac conditions. Otherwise you can use an ICD based approximation of comorbidities (Charlson, Elixhauser) or the PMH section of the discharge summary.
